integration/modules/deploy_helm/values/default.yaml

```diff
@@ -1,8 +1,18 @@
+#Common for telemetry
+agent:
+  config:
+      global:
+        service:
+          telemetry:
+            logging_encoding: json
+            logging_level: INFO
+
+#Common for namepsace/url/token
 observe:
-    collectionEndpoint:
-      value: ${observe_url}
-    token:
-      value: ${observe_token}
+  collectionEndpoint:
+    value: ${observe_url}
+  token:
+    value: ${observe_token}
```

Comment:
This schema is changing pretty rapidly but there's a new top level object `node` as well now

Reply:
@obs-gh-alexlew are you referring to this? 
https://github.com/observeinc/helm-charts/blob/main/charts/agent/values.yaml#L32 

If so, this "default" file is just meant to overwrite anything related to namespace / token / endpoints